lib/rules/lint-quotes.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+/*
+ Enforce quotes style
+
+ ```
+ {{! good }}
+ <img alt="tomster" src="tomster.jpg">
+
+ {{! bad}}
+ <img alt='tomster' src="tomster.jpg">
+ ```
+
+ The following values are valid configuration:
+
+   * boolean -- `false` for disabled. `true` to enforce any quotes
+   * string -- `single` of `double` for enforce style
+ */
+
+var _ = require('lodash');
```

Comment:
[it's possible](https://github.com/oriSomething/lodash-modules-sublime) ;-)
but it means there is no `chain` and lazy evaluation of streams